eng/pipelines/coreclr/runtime-nativeaot-outerloop.yml

```diff
@@ -72,6 +72,7 @@ extends:
                 parameters:
                   creator: dotnet-bot
                   testRunNamePrefixSuffix: NativeAOT_$(_BuildConfig)
+                  extraHelixArguments: /p:IsXUnitLogCheckerSupported=false
```

Comment:
Why not flow `TestNativeAot` down to helix.yml instead of this property?

Reply:
I don't know if it will have any unexpected side effects.